<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>644</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TextEditor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	border: 1px solid;
	border-radius: 5px;
	padding: 10px 20px;
	background-color: white;
}
QLabel{
	font-weight: bold;
	color: white;
}
QPushButton:hover {
	background-color: grey;
}
QPlainTextEdit{
	border: 1px solid white;
	font-size: 20;
	border-radius: 5px;
	padding: 5px 10px;
	background-color: white;
}
QStatusBar {
	color: white;
}
QMenuBar{
	color: white;
	border: 1px solid white;
}
QMenu{
	color: white;
	border: 1px solid white;
}
*{
	background-color: black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="text_editor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>781</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="save_host">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>552</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save To Host</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_client">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>550</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save To Client</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionDelete_File"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionDelete_File">
   <property name="text">
    <string>Delete File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
